docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+---
+Order: 1
+Area: getstarted
+TOCTitle: Tutorial
+ContentId: 72ad9b70-5227-4032-81d7-6aec00a1e8f8
+PageTitle: Getting started with Visual Studio Code
+DateApproved: 08/01/2024
+MetaDescription: This tutorial gives you an overview of the key features of Visual Studio Code to help you get started quickly.
+---
+# Tutorial: Get started with Visual Studio Code
+
+In this tutorial, you learn about the key features of Visual Studio Code to help you get started with coding quickly. You learn about the different components of the user interface and how to customize it to your liking. You then write some code and use the built-in code editing features, such as IntelliSense and Code Actions, and you learn about running and debugging your code. By installing a language extension, you add support for a different programming language.
+
+<iframe src="https://www.youtube.com/embed/B-s71n0dHUk?autoplay=true" width="640" height="320" allowFullScreen="true" frameBorder="0" title="Getting Started with Visual Studio Code"></iframe>
+
+## Prerequisites
+
+- [Download Visual Studio Code](https://code.visualstudio.com/download)
+- [Set up Visual Studio Code on your computer](/docs/setup/setup-overview.md)
```

Comment:
I was confused when I saw this link as I thought this current doc I'm on was the only doc I'd need to start with, and it made me wonder - how much reading does getting started with VS Code require?

When I check out that setup-overview doc, I see it's actually quite concise, and I'd even say I don't think users would have to read it to get started. To avoid users potentially feeling like getting started requires too much reading, I'd remove the setup doc from pre-reqs and maybe just list it in a specific paragraph(s) or troubleshooting section in case users need it for specific issues.

Reply:
Good point.